spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/MetricsProperties.java

```diff
@@ -133,6 +133,20 @@ public void setMaxUriTags(int maxUriTags) {
 			 */
 			private boolean autoTimeRequests = true;
 
+			/**
+			 * Default percentiles when "autoTimeRequests=true" and @Timed annotation is
+			 * not presented on the corresponding request handler. Any @Timed annotation
+			 * presented will have precedence.
+			 */
+			private double[] autoTimeRequestsDefaultPercentiles;
```

Comment:
I am not sure about the `double[]` array, I'd rather use a `List` for this.

Also we now have three properties around the concept of `autoTimeRequests` and some keys are now rather long. Shouldn't we take the opportunity to group those in a group?